src/main/java/duke/command/ByeCommand.java

```diff
@@ -0,0 +1,28 @@
+package duke.command;
+
+import duke.tasklist.TaskList;
+
+import duke.ui.DukeUi;
+
+import duke.storagedata.StorageData;
+
+/**
+ * Represents a Command object where the user input is "bye.
+ */
+public class ByeCommand extends Command {
+    public ByeCommand() {
```

Comment:
I don't think it's necessary to include the `super()` call in the constructor, especially when there is nothing else in the constructor. The `super()` is called by [default](https://stackoverflow.com/questions/2054022/is-it-unnecessary-to-put-super-in-constructor)